src/core/lv2/Lv2Proc.cpp

```diff
@@ -383,6 +508,78 @@ void Lv2Proc::run(fpp_t frames)
 		// Notify the plugin the run() cycle is finished
 		m_worker->notifyPluginThatRunFinished();
 	}
+	
+	// Check if it's time to send updates to the UI
+	m_eventDeltaT += frames;
+	bool sendUiUpdates = false;
+	uint32_t updateFrames = (uint32_t)(Engine::audioEngine()->processingSampleRate() / lv2UiRefreshRate());
```

Comment:
`processingSampleRate()` > `outputSampleRate()`
I had to do this change to make it compile.

Reply:
Good point. Can you append it to this PR with a push, please?

Reply:
OK pushed.